man/PPC-loo.Rd

```diff
@@ -62,10 +62,10 @@ ppc_loo_pit_ecdf(
   ...,
   psis_object = NULL,
   pit = NULL,
-  eval_points = NULL,
+  K = NULL,
   prob = 0.99,
   plot_diff = FALSE,
-  interval_method = c("interpolate", "optimize")
+  interpolate_adj = NULL
```

Comment:
I also unified the arguments for now, to keep the code more simple.
I suggest that, when I work on the documentation and possibly need to rearrange how the PPC function pages are grouped, I will add the clearer argument names and add a deprecation warning to the old ones.

Reply:
Also, with this rearranging, the template could come back to use with `ppc_loo_pit_intervals()`.

It would probably make sense to have `ppc_(loo)_pit_ecdf()`, `ppc_loo_pit_qq()`, and `ppc_loo_pit_overlay()` share a documentation page, and move `ppc_loo_intervals()`, and `ppc_loo_ribbon()` join their LOOless counterparts on the PPC intervals documentation page.

Reply:
> I also unified the arguments for now, to keep the code more simple.
I suggest that, when I work on the documentation and possibly need to rearrange how the PPC function pages are grouped, I will add the clearer argument names and add a deprecation warning to the old ones.

Sounds good, I like that plan.